src/Licenses/LicensesClient.php

```diff
@@ -25,6 +26,11 @@ class LicensesClient extends AbstractLicensesClient
      */
     private const CONFIGS_PATH = '/configs';
 
+    /**
+     * @var string The path of the Predictions endpoint
+     */
+    private const PREDICTION_PATH = '/predictions/daily';
```

Comment:
Why isn't the path just `/predictions`?
I suppose in the future we'll have `/predictions/weekly` or some stuff like that?